snippets/lean-mode/let

```diff
@@ -3,4 +3,5 @@
 # key: let
 # --
 let ${1:this} : $2,
-  by {$3},
+  ${3:by {$4}}
```

Comment:
for this one, we could default to `:=` and put it on the previous line

Reply:
done; I also removed the type ascription since it's not common to use it with `let`